modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebar.es.js

```diff
@@ -57,9 +57,10 @@ class CriteriaSidebar extends Component {
 				<ul className="properties-list">
 					{filteredProperties.length ?
 						filteredProperties.map(
-							({label, name, type}, index) => (
+							({label, name, options, type}, index) => (
 								<CriteriaSidebarItem
 									className={`color--${propertyKey}`}
+									defaultValue={options && options.length ? options[0].value : ''}
```

Comment:
I don't know why github is hiding my comments in deeply nested "hidden item" blocks, but I'm copy pasting this comment here too in case you didn't see it yet.

Instead of having a `defaultValue` being passed into the `CriteriaSidebarItem` (who's only purpose is to pass along the `defaultValue` data), do you think it makes sense to set the default value within `_handleCriterionAdd` in `CriteriaGroup`?  Especially since we're always using the first option value as the default value, I don't think `CriteriaSidebarItem` needs to know about the default value.

Reply:
After thinking about this some more, the `defaultValue` is a good idea. It can make it possible for a default value being set for specific properties.